pkg/convert/pot.go

```diff
@@ -31,6 +33,12 @@ func ToPot(t model.Translation) ([]byte, error) {
 		return nil, fmt.Errorf("write language: %w", err)
 	}
 
+	if !t.Original {
+		if err := writePluralHeader(&b, t); err != nil {
```

Comment:
To simplify everything, we can add Plural headers to non-original files, without checking, if they actually have plurals.
e.g.
```suggestion
		fmt.Fprintf(b, "\"Plural-Forms: nplurals=%d; plural=(n != 1);\\n\"\n\n", pluralCountLimit)
```
Somewhere later, we could adjust that header based on language